FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/multiple_priorities_no_timeslice_utest.c

```diff
@@ -2382,7 +2382,7 @@ void test_task_affinity_resume_suspended_task( void )
  * Task (T0)	             Task (TN)                 Task (TN + 1)
  * Priority – 2              Priority – 1              Priority – 2
  * Affinity – None           Affinity – None           Affinity – None
- * State – Running (Core 0)  State – Running (Core N)  State – Running (Last Core)
+ * State – Running (Core 0)  State – Ready             State – Running (Last Core)
```

Comment:
My thinking on this one was after changing the affinity mask all the TN cores (except 1) are still running, only 1 moves to the ready state. Perhaps there is a better way to display that here?

Reply:
Perhaps we add TN-1 in the comment. It can helps to describe the result here.